ecs-cli/modules/commands/flags.go

```diff
@@ -48,6 +48,7 @@ const (
 	SubnetIdsFlag                   = "subnets"
 	VpcIdFlag                       = "vpc"
 	InstanceTypeFlag                = "instance-type"
+	InstanceRoleFlag                = "instance-role"
```

Comment:
IMO, I think its potentially okay to use "instance-role". It saves the name role in case we want to use it in the future for something- and more importantly, instance role is more descriptive for what it actually does.